bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/BookieInitializationTest.java

```diff
@@ -265,17 +275,23 @@ public void testDuplicateBookieServerStartup() throws Exception {
                 new String[] { tmpDir.getPath() });
         BookieServer bs1 = new BookieServer(conf);
         bs1.start();
-
+        BookieServer bs2 = null;
         // starting bk server with same conf
         try {
-            BookieServer bs2 = new BookieServer(conf);
+            bs2 = new BookieServer(conf);
             bs2.start();
             fail("Should throw BindException, as the bk server is already running!");
         } catch (BindException e) {
-            // Ok
+            Assert.assertTrue("BKServer allowed duplicate startups!",
+                    e.getMessage().contains("Address already in use"));
```

Comment:
this line of code was dropped in 4.4, because e.getMessage().contains depends on the language of the SO, in Italian for instance it says 'Indirizzo già in uso' and the test cannot be run on Italian machines
see https://issues.apache.org/jira/browse/BOOKKEEPER-904